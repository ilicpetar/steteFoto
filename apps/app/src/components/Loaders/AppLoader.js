import React from 'react';
import DashboardLoader from './DashboardLoader';
import PanelLoader from './PanelLoader';
import PanelContent from './PanelContent';

import './Loader.scss';

const AppLoader = () => (
	<div className="Loader">
		<DashboardLoader>
			<PanelLoader>
				<PanelContent />
			</PanelLoader>
		</DashboardLoader>
	</div>
);

export default AppLoader;
