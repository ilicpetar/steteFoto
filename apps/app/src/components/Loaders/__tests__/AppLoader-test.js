import React from 'react';
import { shallow } from 'enzyme';
import AppLoader from '../AppLoader';

describe('AppLoader', () => {
	it('should match snapshot', () => {
		expect(shallow(<AppLoader />)).toMatchSnapshot();
	});
});
