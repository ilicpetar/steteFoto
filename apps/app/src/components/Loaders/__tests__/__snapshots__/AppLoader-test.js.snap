// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoader should match snapshot 1`] = `
<div
  className="Loader"
>
  <DashboardLoader>
    <PanelLoader>
      <PanelContent />
    </PanelLoader>
  </DashboardLoader>
</div>
`;
